app/views/posts/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang=eng>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <%= render 'shared/header'%> 
+      <% unless @posts.empty? %>
+      <% @posts.each do |post| %><br>
+        <div id = "post-box">
+              <span class = "cont">
+                <p > <strong>Author:</strong>    <%= current_user.username %><br></p>
```

Comment:
This is not correct. we want to show the post author not the current user as the owner of every post.

Reply:
Great point sunny I would change it to post.user.username right away